src/components/dashboard/FaceVerification/components/NotMatchError.js

```diff
@@ -0,0 +1,122 @@
+import React, { useCallback, useEffect } from 'react'
+import { View } from 'react-native'
+
+import Text from '../../../common/view/Text'
+import Separator from '../../../common/layout/Separator'
+import { CustomButton, Section, Wrapper } from '../../../common'
+import FaceVerificationErrorSmiley from '../../../common/animations/FaceVerificationErrorSmiley'
+
+import { isMobileOnly } from '../../../../lib/utils/platform'
+import { getDesignRelativeHeight, getDesignRelativeWidth } from '../../../../lib/utils/sizes'
+import { withStyles } from '../../../../lib/styles'
+
+import { fireEvent, FV_NOTMATCHERROR } from '../../../../lib/analytics/analytics'
+
+const NotMatchError = ({ styles, displayTitle, onRetry, nav, exception }) => {
+  const onContactSupport = useCallback(() => nav.navigateTo('Support'), [nav])
```

Comment:
TODO: need design for that screen

Reply:
https://app.zenhub.com/workspaces/gooddollar-5d50833888a83c6880d3e345/issues/gooddollar/gooddapp/2838
opened a separate task for this